src/Header.js

```diff
@@ -11,15 +11,31 @@ import {BrowserRouter as Router, Route, Switch} from "react-router-dom";
 import {css, StyleSheet} from "aphrodite";
 import AppStyles from "./AppStyles";
 import Utils from "./Utils/Utils";
+import Notification from "./Utils/Notification"
 const introJs = require('intro.js/minified/intro.min.js');
 
 class Header extends Component {
+	static lastNotificationTimestamp;
 
 	constructor(props) {
 		super(props);
 		this.state = {
 			hintsVisible: false
 		};
+		Header.lastNotificationTimestamp = new Date();
+	}
+
+	shouldComponentUpdate(nextProps, nextState) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "nextState". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)